src/components/autocomplete-grid/autocomplete-grid-option/autocomplete-grid-option.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { expect } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+
+import { fixture, testA11yTreeSnapshot } from '../../core/testing/private';
+
+import type { SbbAutocompleteGridOptionElement } from './autocomplete-grid-option';
+import '../autocomplete-grid';
+import '../autocomplete-grid-row';
+import './autocomplete-grid-option';
+import '../autocomplete-grid-actions';
+import '../autocomplete-grid-button';
+
+describe('sbb-autocomplete-grid-option', () => {
+  describe('default', () => {
+    let root: SbbAutocompleteGridOptionElement;
+    beforeEach(async () => {
+      root = (
+        await fixture(html`
+          <sbb-autocomplete-grid origin="anchor">
+            <sbb-autocomplete-grid-row>
+              <sbb-autocomplete-grid-option value="1">Option 1</sbb-autocomplete-grid-option>
+            </sbb-autocomplete-grid-row>
+          </sbb-autocomplete-grid>
+          <div id="anchor"></div>
+        `)
+      ).querySelector('sbb-autocomplete-grid-option')!;
+    });
+
+    it('Dom', async () => {
+      await expect(root).dom.to.be.equalSnapshot();
+    });
+
+    it('ShadowDom', async () => {
+      await expect(root).shadowDom.to.be.equalSnapshot();
+    });
+
+    testA11yTreeSnapshot(
+      html`<sbb-autocomplete-grid-option value="1">Option 1</sbb-autocomplete-grid-option>`,
+    );
```

Comment:
You have the double fixture problem here. Move the a11y test outside the describe